packages/types/src/deprecated.ts

```diff
@@ -0,0 +1,29 @@
+export type ModeOption = {
+  id: string;
+  selected?: boolean;
+  showTitle?: boolean;
+  text: JSX.Element;
+  title?: string;
+};
+
+export type ModeSelectorOptions = {
+  primary: ModeOption;
+  secondary?: ModeOption[];
+  tertiary?: ModeOption[];
+};
+
+export type ConfiguredMode =
+  | string
+  | {
+      mode: string;
+      label: string;
+      company?: string;
+    };
+
+export type ConfiguredModes = {
+  transitModes: ConfiguredMode[];
+  accessModes: ConfiguredMode[];
+  exlcusiveModes: ConfiguredMode[];
```

Comment:
Oh I guess the typo was there originally!

Reply:
Haha, did not even notice that in my copy pasta

Reply:
We should fix this